app/src/main/java/org/sopt/official/feature/auth/feature/authmain/AuthMainScreen.kt

```diff
@@ -0,0 +1,303 @@
+            },
+            onFindAccountClick = {
+                navigateToCertification(AuthStatus.SEARCH)
+                loginDialogVisibility = false
+            },
+            onResetAccountClick = {
+                navigateToCertification(AuthStatus.CHANGE)
+                loginDialogVisibility = false
+            },
+            onContactChannelClick = {
+                onContactChannelClick()
+                loginDialogVisibility = false
+            }
+        )
+    }
+
+    LaunchedEffect(viewModel.sideEffect, lifecycleOwner) {
+        viewModel.sideEffect.flowWithLifecycle(lifecycle = lifecycleOwner.lifecycle)
+            .collect { sideEffect ->
+                when (sideEffect) {
+                    is AuthMainSideEffect.ShowToast -> context.toast(sideEffect.message)
+                }
+            }
+    }
+
+    AuthMainScreen(
+        platform = platform,
+        showDialog = {
+            loginDialogVisibility = true
+        },
+        onGoogleLoginCLick = onGoogleLoginCLick,
+        onLoginLaterClick = navigateToUnAuthenticatedHome,
+        navigateToCertification = {
+            navigateToCertification(AuthStatus.REGISTER)
+        }
+    )
+}
+
+@Composable
+private fun AuthMainScreen(
+    platform: String,
+    showDialog: () -> Unit,
+    onGoogleLoginCLick: () -> Unit,
+    onLoginLaterClick: () -> Unit,
+    navigateToCertification: () -> Unit
+) {
+    var showAuthBottom by remember { mutableStateOf(false) }
+    val offsetY = remember { Animatable(0f) }
+
+    LaunchedEffect(Unit) {
+        delay(700)
+        showAuthBottom = true
+        offsetY.animateTo(
+            targetValue = -140f,
+            animationSpec = tween(
+                durationMillis = 1500,
+                easing = FastOutSlowInEasing
+            )
+        )
+    }
+
+    Box(modifier = Modifier.fillMaxSize()) {
+        Column(
+            modifier = Modifier.fillMaxSize(),
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+            Spacer(modifier = Modifier.weight(1f))
+            Image(
+                painter = painterResource(id = R.drawable.img_logo),
+                contentDescription = "솝트 로고",
+                modifier = Modifier.offset(y = offsetY.value.dp)
+            )
+            Spacer(modifier = Modifier.weight(1f))
+        }
+        AnimatedVisibility(
+            visible = showAuthBottom,
+            enter = fadeIn(
+                initialAlpha = 0.1f,
+                animationSpec = tween(easing = EaseIn)
+            ),
+            modifier = Modifier.align(Alignment.BottomCenter)
+        ) {
+            AuthBottom(
+                platform = platform,
+                showDialog = showDialog,
+                onGoogleLoginCLick = onGoogleLoginCLick,
+                onLoginLaterClick = onLoginLaterClick,
+                navigateToCertification = navigateToCertification
+            )
+        }
+    }
+}
+
+@Composable
+private fun AuthBottom(
```

Comment:
하단 영역은 `Footer`라고들 많이하죠?

```suggestion
private fun AuthFooter(
```